### PR TITLE
fix clang format error. remove blck_all collision setting to use with…

### PR DESCRIPTION
**Pull request by yuokamoto**
… InteractiveMarker

### CODE REVIEW

File: Source/RapyutaSimulationPlugins/Private/Core/RRProceduralMeshComponent.cpp [hunk 1]
```diff
@@ -68,27 +68,23 @@ bool URRProceduralMeshComponent::InitializeMesh(const FString& InMeshFileName)
             else
             {
                 URRThreadUtils::DoAsyncTaskInThread<void>(
-                    [this, InMeshFileName]()
-                    {
+                    [this, InMeshFileName]() {
```
Comment (Tadinu) [inline Source/RapyutaSimulationPlugins/Private/Core/RRProceduralMeshComponent.cpp:71]: This seems to make inline `{` only in lambda, but not for if/else, so abit inconsistent but more essentially, just curious why the change?

**Review by Tadinu — APPROVED** [review]

File: Source/RapyutaSimulationPlugins/Private/Core/RRProceduralMeshComponent.cpp (hunk 1 above)
Comment (yuokamoto) [reply Source/RapyutaSimulationPlugins/Private/Core/RRProceduralMeshComponent.cpp:71]: hmm, not sure it was applied by clang-format of vscode.
